src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
```suggestion
    private String ballColor;
    private int ballNumber;
```
Class fields should be hidden from unwanted access - to achieve encapsulation